gymnasium_robotics/envs/multiagent_mujoco/coupled_half_cheetah.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import gymnasium
+import numpy as np
+from gymnasium.envs.mujoco import mujoco_env
+from gymnasium.utils.ezpickle import EzPickle
+
+DEFAULT_CAMERA_CONFIG = {
+    "distance": 4.0,
+}
+
+
+class CoupledHalfCheetah(mujoco_env.MujocoEnv, EzPickle):
+    """This environment consists of 2 half cheetahs coupled by an elastic tendon."""
```

Comment:
This should be a full docstring like the other mujoco environments

Reply:
What do you mean by "full docstring"